### PR TITLE
[Ikanj David]-Sage-Blockchain-Based Festival Booking System with CUSD Payment Integration

### PR DESCRIPTION
**Pull request by Ikanji201**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -733,6 +733,13 @@ const Users: User[] = [
     website: 'blog/tutorials/introducing-celo-sage',
     tags: ['beginner','celosage'],
   },
+  {
+    title: 'Blockchain-Based Festival Booking System with CUSD Payment Integration',
+    description: 'A decentralized platform for event organizers to showcase their programs and allow users to book their slots using cUSD. With Festival, event organizers can write descriptions of their programs including URLs, themes, descriptions, locations, and prices. Users can then browse and book their slots using cUSD tokens.',
```
Comment (RuthO1) [inline src/data-tutorials/users.tsx:738]: reduce the length of the description

Comment (4undRaiser) [inline src/data-tutorials/users.tsx:738]: reduce the length of the description

File: blog/03-28-2023-Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration/Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
+---
+title: Blockchain-Based Festival Booking System with CUSD Payment Integration
+description: A decentralized platform for event organizers to showcase their programs and allow users to book their slots using cUSD. With Festival, event organizers can write descriptions of their programs including URLs, themes, descriptions, locations, and prices. Users can then browse and book their slots using cUSD tokens.
```
Comment (4undRaiser) [inline blog/03-28-2023-Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration/Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration.md:3]: reduce here too

File: blog/03-28-2023-Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration/Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration.md [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+  - name: David Ikanji
+    title: Technical Writer 
+    url:  https://github.com/Ikanji201
+    image_url:  https://avatars.githubusercontent.com/u/115812158?v=4
+tags: [solidity, intermediate, celo, celosage]
+hide_table_of_contents: true
+slug: /tutorials/blockchain-based-festival-booking-system-with-cusd-payment-integration
+---
+
+![Blockchain-Based Festival Booking System with CUSD Payment Integration](https://user-images.githubusercontent.com/115812158/228259160-bca6cc82-4d0e-4de8-9ea0-569dc9f9e2ec.png)
```
Comment (4undRaiser) [inline blog/03-28-2023-Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration/Blockchain-Based-Festival-Booking-System-with-CUSD-Payment-Integration.md:14]: This is not the right format for header image, you should import it it from showcase folder

**Comment by 4undRaiser** [comment]
@Ikanji201 your pr is not compiling. fix your image name and also the format for inporting it to your md file, do that and deploy again let's see.

image name should be all lower case seperated by a dash

**Review by Ishanpathak1 — APPROVED** [review]
Approved @developerkunal

**Pull request merged**